src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -329,6 +330,15 @@ public void setPipeline(Pipeline pipeline) {
         this.pipeline = pipeline;
     }
 
+    public Pipeline getHeadPipeline() {
+        return headPipeline;
+    }
+
+    public MergeRequest setHeadPipeline(Pipeline headPipeline) {
+        this.headPipeline = headPipeline;
+        return this;
+    }
```

Comment:
In this project the setters are returning `void`.

When an attribute is used to create or update an entity, there is a builder method `withHeadPipeline(..)` next to the setter.
But in this case the `head_pipeline` attribute is only used during a read, so the builder method makes no sense.

```suggestion
    public void setHeadPipeline(Pipeline headPipeline) {
        this.headPipeline = headPipeline;
    }
```